src/components/Tabs/Tabs.js

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from "react";
+import Tab from "./Tab";
+import StyledTabs from "./Tabs.styled";
+
+const Tabs = (props) => {
+  const initActiveTab = "home";
+  const [activeTab, setActiveTab] = useState("home");
+
+  const handleTabChange = (newActiveTab) => {
+    setActiveTab(newActiveTab);
+  };
+
+  const tabList = props.children.map((tab, index) => {
+    const {
+      props: { title, eventKey, active, disabled },
+    } = tab;
+    return (
+      <Tab
+        key={index}
+        handleTabChange={handleTabChange}
+        active={active}
+        eventKey={eventKey}
+        title={title}
+        disabled={disabled}
+      ></Tab>
+    );
+  });
+
+  const renderTabContent = () => {
+    // tutaj jest PROBLEM z propsami bo muszę się przebijać przez wszsytko
```

Comment:
Przy takie strukturze tak to wygląda.
Zobacz oryginalny komponent:
https://github.com/react-bootstrap/react-bootstrap/blob/master/src/Tabs.tsx